Solutions/Recorded Future/Playbooks/RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor/RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor.json

```diff
@@ -3,11 +3,11 @@
     "contentVersion": "1.0.0.0",
     "metadata": {
         "title": "RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor",
-        "description": "This playbook leverages the Recorded Future API and automatically imports the C&C DNS Name Domain RiskList, as Threat Intelligence Indicators, for detection purposes in Microsoft Sentinel",
+        "description": "This playbook leverages the Recorded Future API and automatically imports the C&C DNS Name Domain RiskList, as Threat Intelligence Indicators, for detection purposes in Microsoft Sentinel.\n\nThis playbook depends on RecordedFuture-ImportToSentinel that need to be installed **manually** before installing current playbook.",
         "prerequisites": [ "None" ],
```

Comment:
@RecordedFutureOskbo 

As all the playbooks, required recorded future v2 custom connector pre installed and api key is needed as well, Can you please add that in prerequisites .  

and add one more field which is "prerequisitesDeployTemplateFile":

For Example , Please check the below playbook.

https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-Enrichment/azuredeploy.json

Kindly do this for all playbooks

Reply:
I have updated the playbook documentation. Recorded FutureV2 connector is published in power platform and does not have to be referenced as a "prerequisitesDeployTemplateFile" if I'm correct?
https://learn.microsoft.com/en-us/connectors/recordedfuturev2/